addons/website/static/src/snippets/s_tabs/options.js

```diff
@@ -44,7 +52,7 @@ options.registry.NavTabs = options.registry.MultipleItems.extend({
      */
     _findLinksAndPanes: function () {
         this.$navLinks = this.$target.find('.nav:first .nav-link');
```

Comment:
This line shouldn't be adapted either? And perhaps check all the other selectors of the class...